utils/generic-components.stanza

```diff
@@ -266,6 +176,41 @@ public defn look-up-ceramic-caps (attribute: String, filter-properties:Tuple<Key
                                         filter-properties]
   look-up-capacitors(attribute, params)
 
+; ========================================================
+; ==================== tantalum-cap =======================
+; ========================================================
+
+public defn tantalum-cap (user-params:Tuple<KeyValue>) -> Instantiable :
+  val params = remove-duplicate-keys $ [["anode" => "tantalum" "min-rated-voltage" => MIN-CAP-VOLTAGE]
+                                        smd-query-properties()
+                                        user-params]
+  to-jitx $ Capacitor(params)
+
+public defn tantalum-cap (cap:Double) -> Instantiable :
+  tantalum-cap(["capacitance" => cap])
+
+public defn tantalum-cap (cap:Double, tolerance:Double) -> Instantiable :
+  tantalum-cap(["capacitance" => cap "tolerance" => tolerance])
+
+public defn tantalum-caps (user-params:Tuple<KeyValue>, limit: Int) -> Tuple<Instantiable> :
+  val params = remove-duplicate-keys $ [["anode" => "tantalum" "min-rated-voltage" => MIN-CAP-VOLTAGE]
+                                        smd-query-properties()
+                                        user-params]
+  map(to-jitx, Capacitors(params, limit))
+
+; ========================================================
+; =============== look-up-tantalum-caps ===================
+; ========================================================
+
+public defn look-up-tantalum-caps (attribute: String) -> Tuple :
```

Comment:
What are the valid values for the `attribute` argument and what does it control? 

What is the return of this? Is it `Tuple<JSON>`?

Reply:
pass - this gets captured in our docs

Reply:
I don't see any docs for this.